doc/manual/parallel-computing.rst

```diff
@@ -170,15 +170,19 @@ the following code::
 Starting Julia with ``julia -p 2``, you can use this to verify the following:
 
 - :func:`include("DummyModule.jl") <include>` loads the file on just a single process (whichever one executes the statement).
-- ``using DummyModule`` causes the module to be loaded on all processes; however, the module is brought into scope only on the one executing the statement.
+- ``@everywhere import DummyModule`` causes the module to be loaded on all processes. However, it is not brought into scope on
```

Comment:
is the module not brought into scope, or is this referring to the exports of the module (using vs import distinction) ?